frontend/lib/pages/loc-confirmation.tsx

```diff
@@ -34,7 +34,7 @@ const SanitationGuidelines = () => {
               {!inSafeMode && 
                 <>{' '}<button className={classnames("button","is-text","is-paddingless","is-uppercase", isExpanded && "is-hidden")} role="button" 
                     onClick={() => toggleExpansion(true)} aria-label="Show me sanitation guidelines" aria-expanded={ariaBool(isExpanded)}>
-                Learn More
+                Show More
```

Comment:
Hey can you change this to "Show more"?  We're trying to use [sentence case](https://medium.com/@jsaito/making-a-case-for-letter-case-19d09f653c98) so we like "Show more" over "Show More" if possible.

Oops I'm realizing that I should've noticed this on your previous PRs!  Sorry about that.  Also, since I think the styling of this text is set to make it all-uppercase anyways, it's probably not that big a deal, but for consistency's sake I think it'd be good to bust it up in sentence case.

Reply:
Ugh good catch— I have a major bad habit of over-capitalizing everything I write... seems it's slipped in here too